attributes/default.rb

```diff
@@ -43,6 +43,14 @@
 }
 default['consul']['source_revision'] = 'master'
 
+# GOMAXPROCS should be greater than 1, see
+# https://groups.google.com/forum/#!topic/consul-tool/qewFEqgAoF8
+if node['cpu']['total'] > 1
+  default['consul']['gomaxprocs'] = node['cpu']['total']
```

Comment:
Well, the only thing I wanted to change is that it isn't set to 1 by default, which (according to the mailing list thread in the comment) isn't completely safe.